Dockerfile

```diff
@@ -50,6 +50,7 @@ RUN yarn --frozen-lockfile --non-interactive --ignore-optional
 # Build common
 WORKDIR /app/packages/common/
 COPY packages/common/ .
+COPY tsconfig.json /app
```

Comment:
This step does not belong to the build common so I would move it a few lines up

Reply:
True, fixing it